routes/index.js

```diff
@@ -1,9 +1,16 @@
 var express = require('express');
 var router = express.Router();
+const Projects = require('../database/db.js')
```

Comment:
if you change the export for the db file then you can do destructure this import
```
const { Projects } = require('../database/db.js')
```